plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -233,6 +234,16 @@ class NewCreateServiceModalForm extends Component {
     this.setState({batch, appConfig: this.getAppConfig(batch)});
   }
 
+  handleSetItem({value, path}) {
```

Comment:
My reasoning was that `ADD_ITEM` and `REMOVE_ITEM` refer to arrays, but you are actually right. These are just semantics and the reducer can decide how to handle them.

Fixed 👍